README.md

```diff
@@ -0,0 +1,39 @@
+
+# 📖 [Hello World Project] <a name="about-project"></a>
+This is the first Microverse hello world project which does not require any prior coding experience.
+
+## 🛠 Built With <a name="built-with"></a>
+
+<strong>HTML</strong><br>
+<strong>CSS</strong>
+### Key Features <a name="key-features"></a>
+
+- **[Headline]**
+- **[Homepage]**
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/Kevin-Mena)
+- Twitter: [@twitterhandle](https://twitter.com/Fmenawende)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kevin-okoth-19407119b/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Navigation bar]**
+-  **[Visual support]**
+- **[Footer]**
+
```

Comment:
- [ ] Please do not delete any section of the microverse template as I see you did that, but rather, kindly edit all the sections of your readme and add information relating to your project so that your repo will look more professional 👍🏽. Your readme is missing the acknowledgment and the get started section